flink/v1.12/arctic-flink-iceberg-bridge/pom.xml

```diff
@@ -0,0 +1,561 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>arctic-parent</artifactId>
+        <groupId>com.netease.arctic</groupId>
+        <version>0.5.0-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>arctic-flink-iceberg-bridge</artifactId>
+    <name>Arctic Project Flink 1.12 opensource iceberg</name>
```

Comment:
Capitalization is required for module name like `Arctic Project Flink 1.12 Iceberg Bridge`